src/Tizen.NUI/src/public/VisualMaps.cs

```diff
@@ -661,17 +661,21 @@ internal void UpdateVisual(bool force = false)
         /// <since_tizen> 3 </since_tizen>
         protected virtual void ComposingPropertyMap()
         {
-            if (null == _outputVisualMap)
-            {
-                _outputVisualMap = new PropertyMap();
-            }
+            _outputVisualMap = new PropertyMap();
 
             if (_shader != null) { _outputVisualMap.Add(Visual.Property.Shader, new PropertyValue(_shader)); }
             if (_premultipliedAlpha != null) { _outputVisualMap.Add(Visual.Property.PremultipliedAlpha, new PropertyValue((bool)_premultipliedAlpha)); }
             if (_mixColor != null) { _outputVisualMap.Add(Visual.Property.MixColor, new PropertyValue(_mixColor)); }
             if (_opacity != null) { _outputVisualMap.Add(Visual.Property.Opacity, new PropertyValue((float)_opacity)); }
             if (_visualFittingMode != null) { _outputVisualMap.Add(Visual.Property.VisualFittingMode, new PropertyValue((int)_visualFittingMode)); }
             if (_cornerRadius != null) { _outputVisualMap.Add(Visual.Property.CornerRadius, new PropertyValue((int)_cornerRadius)); }
+
+            var transformMap = OutputTransformMap;
+
+            if (!transformMap.Empty())
```

Comment:
very good to empty check! (b^_^)b

Reply:
One question.
Is there any reason that we cannot use "OutputTransformMap" directly instead of creating  "transformMap" ?

Reply:
In the next line, transformMap is re-used.
```C#
var transformMap = OutputTransformMap;

if (!transformMap.Empty())
{
    _outputVisualMap.Add(Visual.Property.Transform, new PropertyValue(transformMap));
}
```
If it use `OutputTransformMap` property directly, it calls property getter twice and that leads creating transform map twice redundantly.

Reply:
I understand. Thanks for answering.